API.md

```diff
@@ -382,15 +357,16 @@ by gltf sample viewer
 **Kind**: static property of [<code>GltfState</code>](#GltfState)  
 
 * [.ToneMaps](#GltfState.ToneMaps)
-    * [.NONE](#GltfState.ToneMaps.NONE)
+    * [.KHR_PBR_NEUTRAL](#GltfState.ToneMaps.KHR_PBR_NEUTRAL)
     * [.ACES_HILL_EXPOSURE_BOOST](#GltfState.ToneMaps.ACES_HILL_EXPOSURE_BOOST)
     * [.ACES_NARKOWICZ](#GltfState.ToneMaps.ACES_NARKOWICZ)
     * [.ACES_HILL](#GltfState.ToneMaps.ACES_HILL)
+    * [.NONE](#GltfState.ToneMaps.NONE)
 
-<a name="GltfState.ToneMaps.NONE"></a>
+<a name="GltfState.ToneMaps.KHR_PBR_NEUTRAL"></a>
 
-#### ToneMaps.NONE
-don't apply tone mapping
+#### ToneMaps.KHR\_PBR\_NEUTRAL
```

Comment:
Yes, 3D Commerce officially voted the name as Khronos PBR Neutral Tone Mapping, or PBR Neutral for short.